plex_generate_previews.py

```diff
@@ -71,8 +134,10 @@ def generate_images(video_file, output_folder, lock):
         speed = speed[-1]
     logger.info('Generated Video Preview for {} HW={} TIME={}seconds SPEED={}x '.format(video_file, hw, seconds, speed))
 
-    # Rename images
+    # Optimize and Rename Images
     for image in glob.glob('{}/img*.jpg'.format(output_folder)):
+        im = Image.open(image)  
```

Comment:
@jasonsansone when i tested i tried to mimic how plex handled it. When i ran a side by side comparison of the filesize. The bif was the same using this script or plex.

I would have thought ffmpeg would already compress the image so is this needed?